content/en/docs/collector/getting-started.md

```diff
@@ -113,14 +113,14 @@ systems. The packaging includes a default configuration that can be found at
 
 > Please note that systemd is require for automatic service configuration
 
-To get started on Debian systems run the following replacing `v0.38.0` with the
+To get started on Debian systems run the following replacing `v0.39.0` with the
 version of the Collector you wish to run and `amd64` with the appropriate
 architecture.
 
 ```console
 $ sudo apt-get update
 $ sudo apt-get -y install wget systemctl
-$ wget https://github.com/open-telemetry/opentelemetry-collector-releases/releases/download/v0.38.0/otelcol_0.38.0_linux_amd64.deb
+$ wget https://github.com/open-telemetry/opentelemetry-collector-releases/releases/download/v0.39.0/otelcol_0.39.0_linux_amd64.deb
 $ dpkg -i otelcol_0.38.0_linux_amd64.deb
```

Comment:
There are three places where you'll need to complete the search-and-replace of 0.38 -> 0.39 on lines L124, L127 and L135.